project/SparkBuild.scala

```diff
@@ -1107,10 +1152,10 @@ object Unidoc {
 
     (ScalaUnidoc / unidoc / unidocProjectFilter) :=
       inAnyProject -- inProjects(OldDeps.project, repl, examples, tools, kubernetes,
-        yarn, tags, streamingKafka010, sqlKafka010, connect),
+        yarn, tags, streamingKafka010, sqlKafka010, connect, protobuf),
```

Comment:
Same here: Is this required? Wasn't needed for Avro. I don't have much context here. cc: @gengliangwang, @SandishKumarHN

Reply:
@rangadi I was following the connector/connect module for shading jars and thought this is needed. Based on [sbt-unidoc](https://github.com/sbt/sbt-unidoc) it skips Scaladoc for the protobuf project. not sure if we need this or not.